# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/QuotaParent.h
# Commit: b12a2f55233f
# Full Hash: b12a2f55233fade8f2e3411295ff3d362df9e359
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-09 21:46:09
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Change nsIQuotaManagerService::ResetStoragesForPrincipal to support resetting of entire origins only; r=dom-storage-reviewers,jari
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195510
# ==============================================================================

diff -r 6a90f36c8bf5 -r b12a2f55233f dom/quota/QuotaParent.h
--- a/dom/quota/QuotaParent.h	Wed Oct 09 11:59:34 2024 +0000
+++ b/dom/quota/QuotaParent.h	Wed Oct 09 11:59:35 2024 +0000
@@ -111,7 +111,7 @@
 
   virtual mozilla::ipc::IPCResult RecvShutdownStoragesForOrigin(
       const Maybe<PersistenceType>& aPersistenceType,
-      const PrincipalInfo& aPrincipalInfo, const Maybe<Type>& aClientType,
+      const PrincipalInfo& aPrincipalInfo,
       ShutdownStoragesForOriginResolver&& aResolve) override;
 
   virtual mozilla::ipc::IPCResult RecvShutdownStoragesForClient(